RELEASE/scripts/autoscend/paths/auto_path_util.ash

```diff
@@ -3,7 +3,12 @@
 boolean auto_buySkills()  // This handles skill acquisition for general paths
 {
 	// TODO: Torso Awareness is worth obtaining in other cases too.
-	if((my_meat() >= 10000)
+	int torsoPrice = 10000;
+	if(get_property("auto_torsoAcquirePrice").to_int() != 0)
+	{
+		torsoPrice = get_property("auto_torsoAcquirePrice").to_int();
+	}
```

Comment:
instead of this, just set a default value in auto_setting.ash so that there is always a value set.
look at void auto_settingsDefaults()